saleor/static/dashboard-next/orders/views/OrderDetails.tsx

```diff
@@ -337,264 +338,278 @@ export const OrderDetails: React.StatelessComponent<OrderDetailsProps> = ({
-                            onOrderLineUpdate={handleLineUpdate}
-                            onOrderFulfillmentCancel={handleFulfillmentCancel}
-                            onOrderFulfillmentUpdate={handleFulfillmentUpdate}
-                            onDraftFinalize={handleDraftFinalize}
-                            onDraftCancel={handleOrderDraftCancel}
-                            onOrderMarkAsPaid={handleMarkAsPaid}
-                          >
-                            {({
-                              errors,
-                              orderAddNote,
-                              orderCancel,
-                              orderCreateFulfillment,
-                              orderDraftUpdate,
-                              orderLineAdd,
-                              orderLineDelete,
-                              orderLineUpdate,
-                              orderPaymentCapture,
-                              orderPaymentRefund,
-                              orderVoid,
-                              orderShippingMethodUpdate,
-                              orderUpdate,
-                              orderFulfillmentCancel,
-                              orderFulfillmentUpdateTracking,
-                              orderDraftCancel,
-                              orderDraftFinalize,
-                              orderPaymentMarkAsPaid
-                            }) =>
-                              maybe(
-                                () => order.status !== OrderStatus.DRAFT
-                              ) ? (
-                                <OrderDetailsPage
-                                  errors={errors}
-                                  onNoteAdd={variables =>
-                                    orderAddNote.mutate({
-                                      input: variables,
-                                      order: id
-                                    })
-                                  }
-                                  onBack={() => navigate(orderListUrl)}
-                                  order={order}
-                                  countries={maybe(
-                                    () => data.shop.countries,
-                                    []
-                                  ).map(country => ({
-                                    code: country.code,
-                                    label: country.country
-                                  }))}
-                                  shippingMethods={maybe(
-                                    () => data.order.availableShippingMethods,
-                                    []
-                                  )}
-                                  onOrderCancel={variables =>
-                                    orderCancel.mutate({
+                          <>
+                            <WindowTitle
+                              title={maybe(() => "#" + data.order.number)}
```

Comment:
I think "Order #{number}" would be better here.

Reply:
Also, I think it might be useful to display "Draft order #{number}" for drafts.

Reply:
Sure, on it